### DISCUSSION TITLE
Multiple spouses without connection crossing

### DISCUSSION THREAD

**Discussion by codersclub** [Serg-Norseman/GEDKeeper#506] [Ideas]
In real life, families with several spouses are very common.
Therefore, I would really like GedKeeper to be able to correctly display family trees with multiple spouses.
Here's a real life example:
![multispouse](https://github.com/Serg-Norseman/GEDKeeper/assets/3078296/03a95310-526c-4b4f-b688-d773ec7abdd7)

However, GedKeeper does not allow you to see such a picture.
GedKeeper displays this tree as follows:
![gedkeeper2](https://github.com/Serg-Norseman/GEDKeeper/assets/3078296/2e8795bc-dc64-4113-a540-71ddcd99575a)

![gedkeeper3](https://github.com/Serg-Norseman/GEDKeeper/assets/3078296/61780d10-e841-43fe-8045-5c49a23191cd)

It would be very cool if GedKeeper could draw such a tree correctly!
I have attached a test gedcom file below.

[Family-8.ged.gz](https://github.com/Serg-Norseman/GEDKeeper/files/12784814/Family-8.ged.gz)

**Comment by Serg-Norseman** [comment]
I will think. But right now, unfortunately, my engine is not flexible enough.

**Comment by codersclub** [comment]
Ok, then let me suggest yet another real family tree for your inspiration :)
[Ottoman family tree](https://en.wikipedia.org/wiki/Ottoman_family_tree)
